CoroutineTemplate/build.gradle.kts

```diff
@@ -35,8 +35,12 @@ tasks.withType<io.gitlab.arturbosch.detekt.Detekt>().configureEach {
 }
 
 detekt {
-    toolVersion = Versions.DETEKT_VERSION
```

Comment:
Hmmmm, so we no longer need this? 😁 

Seems like it's still mentioned in their [docs](https://detekt.dev/)

Reply:
@Tuubz I just checked, `toolVersion` belongs to `CodeQualityExtension`, the parent of `DetektExtension` from `detekt` config block. Reverted ✅